src/assets.ts

```diff
@@ -379,13 +399,13 @@ interface PromptItem extends vscode.MessageItem {
     result: PromptResult;
 }
 
-function promptToAddAssets() {
+function promptToAddAssets(workspaceFolder: vscode.WorkspaceFolder) {
```

Comment:
Does this create the tasks.json and launch.json for the multi-root workspace?

Reply:
No, this causes us to prompt for the correct workspace folder on which OmniSharp is launched. OmniSharp itself does not support multiple roots. It can only be launched on one thing at a time, which can be switched by clicking the OmniSharp status bar item in VS Code. So, if OmniSharp is launched in a different folder other than the first one, we need to ensure that we generate in that workspace folder. Make sense?

Reply:
Sounds good!